bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -212,6 +216,12 @@ type BottlerocketKubernetesSettings struct {
 	ClusterDNSIPs []string `json:"clusterDNSIPs,omitempty"`
 }
 
+// BottlerocketKernelSettings holds the kernel settings for bottlerocket nodes
+type BottlerocketKernelSettings struct {
+	// SysctlSettings defines the kernel sysctl settings to set for bottlerocket nodes.
+	SysctlSettings map[string]string `json:"sysctlSettings,omitempty"`
```

Comment:
Why can't this be structured like the kubernetes settings?

Reply:
The kubernetes settings set the list it takes in as the value for the `allowed-unsafe-sysctls` key. The kernel settings takes in key/value pairs and gets set like this instead:
```
[settings.kernel.sysctl]
"key" = "value"
```
So we need to structure it in a way to pass in both key and value